src/main/scala/castalia/actors/JsonResponsesEndpointActor.scala

```diff
@@ -28,6 +29,8 @@ class JsonResponsesEndpointActor(override val endpoint: String, val responses: L
   import context._

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/scala-academy/Castalia/file/2016772517/issues/source?bid=2998213&fileBranchId=3123962#l29)